gnocchi/storage/__init__.py

```diff
@@ -682,6 +682,12 @@ def _map_compute_splits_operations(bound_timeserie):
 
             new_boundts.append((metric, ts.serialize()))
 
+            # If the archive policy backwindow is changed, the data is going to
+            # be truncated in the processing of new datapoints. Therefore, we
+            # can mark the metric as not needing raw data truncation anymore
+            if metric.needs_raw_data_truncation:
+                indexer_driver.update_needs_raw_data_truncation(metric.id)
+
```

Comment:
This is not on every call. It will only update the field if the "needs_raw_data_truncation" is set to true, which will only happens when the backwindow is changed, and only when it is reduced.

Reply:
sorry, misread this.

Reply:
No worries, it happens. Again, thank you so much for the support in reviewing the patches!